src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
suggestion: I understand the purpose of creating this file, but if you don't need it anymore – remove it